service/src/msr_data_skx.cpp

```diff
@@ -61,7 +61,8 @@ namespace geopm
                     "units":     "none",
                     "scalar":    1,
                     "behavior":  "constant",
```

Comment:
Realized this is no longer "constant" when writable. I need to make it variable instead.